code/datums/datumvars.dm

```diff
@@ -78,7 +78,7 @@
 
 /client/proc/debug_variables(datum/D in world)
 	set category = "Debug"
-	set name = "View Variables"
+	set name = "\[Admin\] View Variables"
```

Comment:
This offsetting things in the verbs panel is kinda ehh

Reply:
Yeah, I can't really think of a way around this because of how these bloody verbs work. I didn't want to remove them from the verb tab since some admins might use them there for whatever reason.